contracts/keep3r-liquidity-manager/Keep3rLiquidityManagerUserJobsLiquidityHandler.sol

```diff
@@ -126,7 +135,7 @@ abstract contract Keep3rLiquidityManagerUserJobsLiquidityHandler is
     // substract amount from user idle amount
     userLiquidityIdleAmount[_user][_lp] = userLiquidityIdleAmount[_user][_lp].sub(_amount);
     // add lp to job if that lp was not being used on that job
-    if (jobLiquidityDesiredAmount[_job][_lp] == 0) _addLPToJob(_lp, _job);
+    if (jobLiquidityDesiredAmount[_job][_lp] == 0) _addLPToJob(_lp, _job); // TODO check if already added (can be added and desired)
```

Comment:
`can be added and desired`, what ? :yum:

Reply:
Comment can be removed since `_addLPToJob` only adds unique LPs and does not throw.
This was because an LP can be active even if desired amount is 0.
- escrow1 add liq (desired 10)
- user removes liq (desired 0, but job still has liquidity enabled)
- user adds liquidity again (desired 0 -> 10, code will try to re-add liquidity)